src/msg_data.c

```diff
@@ -117,6 +117,57 @@ u8 MsgCharWidthNormal[] = {
     [MSG_CHAR_UPPER_I_UMLAUT]         8,
     [MSG_CHAR_UPPER_N_TILDE]          9,
     [MSG_CHAR_UPPER_O_GRAVE]          9,
+#if VERSION_IQUE
+    [MSG_CHAR_UPPER_O_ACUTE]         18,
```

Comment:
Can we use `[MSG_CHAR_UPPER_O_ACUTE ... MSG_CHAR_BUTTON_C_LEFT] = 18,` here? Or does that not work for our compiler?